lms/static/certificates/sass/_components.scss

```diff
@@ -537,7 +537,7 @@
 // #FOOTER
 // ------------------------------
 .footer-app {
-    border-top: rem(4) solid palette(grayscale, xx-trans);
+    border-top: rem(4) solid transparentize(palette(grayscale, base), 0.95);
```

Comment:
Depending on the background below an element changing the transparency can alter the element's shade. I can see how this could be a useful way of defining a range of colours based on a single input colour, but I think we should only be using transparency when we want to be able to see through an element. Otherwise we cannot be sure what shade the colour will render as (2 of the same elements would be different shades if added to 2 different containers with different backgrounds).